internal/authz/repository/user_grant.go

```diff
@@ -1,12 +1,7 @@
 package repository
 
-import (
-	"context"
-
-	"github.com/caos/zitadel/internal/api/authz"
-)
+import "context"
 
 type UserGrantRepository interface {
-	ResolveGrants(ctx context.Context) (*authz.Grant, error)
 	SearchMyZitadelPermissions(ctx context.Context) ([]string, error)
```

Comment:
cant we move this request to query side?

Reply:
i think you're right... found some more